splinterd/tests/framework/network.rs

```diff
@@ -78,7 +78,7 @@ impl Network {
         self
     }
 
-    pub fn node(self: &mut Network, n: usize) -> Result<&mut Node, InvalidArgumentError> {
+    pub fn node_mut(self: &mut Network, n: usize) -> Result<&mut Node, InvalidArgumentError> {
```

Comment:
I think we can just change this method to not be `mut`, since it's currently not necessary (which would not require the commit that makes use of it just for the sake of using it). Easy enough to add back.

Reply:
Updated to remove the `node_mut` method and the extra commit that used this method.